geode-core/src/main/java/org/apache/geode/internal/security/SecurableCommunicationChannel.java

```diff
@@ -25,25 +27,14 @@
   WEB(SecurableCommunicationChannels.WEB),
   GATEWAY(SecurableCommunicationChannels.GATEWAY),
   LOCATOR(SecurableCommunicationChannels.LOCATOR),
-  NONE("NO_COMPONENT");
+  NONE("none");
```

Comment:
This should be made a constant, `SecurableCommunicationChannels.NONE`, to be consistent with the other declarations.